scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -283,8 +283,20 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         }
     }
 
+    private fun getFileCount(result: JsonNode): Int {
+        // Handling for ScanCode 2.9.8+
```

Comment:
To ensure the `+` is not read as a part of the version, maybe simply write it out: "Handling for ScanCode 2.9.8 and above."